build/Targets/Imports.targets

```diff
@@ -20,21 +21,21 @@
         <OutDir>$(OutDir)UnitTests\$(MSBuildProjectName)\</OutDir>
       </PropertyGroup>
     </When>
-    <When Condition="'$(RoslynProjectType)' == 'UnitTestPortable' or '$(RoslynProjectType)' == 'UnitTestDesktop'">
+    <When Condition="'$(RoslynProjectType)' == 'UnitTestPortable' OR '$(RoslynProjectType)' == 'UnitTestDesktop'">
       <!-- Portable DLLs don't have a supported mechanism for deploying their dependencies
-           as a part of a normal build.  This breaks developer workflows like F5, TDD, and 
+           as a part of a normal build.  This breaks developer workflows like F5, TDD, and
            standard unit testing.
 
            For developer convenience we use an unsupported way of deploying dependencies.  This
            includes NuGet and standard references.  This is not done on official builds because
            it's not a supported mechanism hence we don't rely on it for official validation.
-           
-           This bug tracks getting an official solution out of MSBuild.  
+
+           This bug tracks getting an official solution out of MSBuild.
 
            https://github.com/Microsoft/msbuild/issues/1499 -->
       <PropertyGroup>
-        <_CopyReferences>$(DeveloperBuild></_CopyReferences>
-        <_CopyProjectReferences>$(DeveloperBuild></_CopyProjectReferences>
+        <_CopyReferences>$(DeveloperBuild)</_CopyReferences>
```

Comment:
@jaredpar, the closing parentheses was not an actual parentheses. This may have impacted some stuff